--- OpenWrt-ImageBuilder-ar71xx_generic-for-linux-x86_64/include/image.mk.old	2016-02-11 13:20:34.145864595 -0600
+++ OpenWrt-ImageBuilder-ar71xx_generic-for-linux-x86_64/include/image.mk	2016-02-11 13:20:16.694865635 -0600
@@ -115,6 +115,13 @@
 ifneq ($(CONFIG_TARGET_ROOTFS_TARGZ),)
   define Image/mkfs/targz
 		# Preserve permissions (-p) when building as non-root user
+		#
+		# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+		# Going to pause now just before tarring up the rootfs.
+		# Make mods to the rootfs now, then when you're done, press <Enter>
+		# The directory contain the rootfs is:
+		# $(TARGET_DIR)
+		read
 		$(TAR) -czpf $(BIN_DIR)/$(IMG_PREFIX)-rootfs.tar.gz --numeric-owner --owner=0 --group=0 -C $(TARGET_DIR)/ .
   endef
 endif
